src/components/utils/test/message_queue_test.cc

```diff
@@ -209,7 +209,7 @@ TEST_F(MessageQueueTest,
   threads::Thread* thread1 =
       threads::CreateThread("test thread", new ShutDownQueueDelegate(job_sync));
   thread1->start();
-  job_sync.WaitJobDone(100);
+  job_sync.WaitJobDone(2000);
```

Comment:
@LuxoftAKutsan, in previous commit it has different meaning.
2s now means that UT will wait here up to 2s before give up (and go to check the result).
Usually UT speed is much more faster and no delays are observed.